### PR TITLE
first cut at spec for governance dashboard needs

### PR DESCRIPTION
**Pull request by levity**

### CODE REVIEW

**Comment by levity** [comment]
attn @jparklev, lemme know how this looks

File: gov.graphql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+enum ProposalStatus {
+  DISCUSSION
+  WITHDRAWN
+  VOTING_STARTED
+  VOTING_ENDED
+}
+
+type ProposalOption {
+  id: Int
+  label: String
```
Comment (jparklev) [inline gov.graphql]: Is `label` something you're thinking we should have on-chain? Or should it be fetched from the CMS in this case?

File: gov.graphql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+type Vote {
+  address: Address
+  balance: Float
+}
+
+type Query {
+  # when blockNumber is specified, the votes and vote balances returned should
+  # match the state at that block. when it is not specified, it should default
```
Comment (jparklev) [inline gov.graphql:30]: What we could do: when a vote has reached a `VOTING_ENDED` state, all child vote balances should, for all future blocks, return what they were for that proposal at it's ending block (& same for all of its ProposalOptions' `voteBalanceSum`).

Comment (levity) [reply gov.graphql:30]: ah, yes, that makes more sense. so the default is the latest block or the block at which the vote ended, whichever is earlier.

**Comment by levity** [comment]
Thx for merging. What’s the process for getting this through the
development backlog? Is there a JIRA issue we can follow keep up to date
about its status?

**Pull request merged**